core/soc/estimator.go

```diff
@@ -107,6 +119,7 @@ func (s *Estimator) SoC(chargedEnergy float64) (float64, error) {
 	if fetchedSoC == nil {
 		f, err := s.vehicle.SoC()
 		if err != nil {
+			// required for online APIs with refresh tokens
```

Comment:
ein refresh token ist egtl etwas anderes

Reply:
Also `// required for online APIs with refreshkey` ?